docs/_chapters/230-manifest-annotations.md

```diff
@@ -1,136 +1,286 @@
 ---
 order: 230
-title: Manifest Annotations
+title: Bundle Annotations
 layout: default
 ---
 
-Entering manifest headers is error prone for a number of reasons since these headers are often complex, have different often hard to remember fields, and are singletons. Because they are singletons, there is only one place where they can be entered: the manifest. This is in contrast with the promoted component model where a component is stand-alone. Components should be cheap and easy to rename or move between bundles. This works fine for its Java imports but if you have an associated header in the manifest then it is easy to forget to also move its corresponding headers from the manifest. This can cause orphaned headers or missing headers either in the old bundle or the new bundle. The old bundle can miss the header because multiple components were depending on that header but it was mistakingly removed.
+Manifest headers are notoriously hard to write. It often takes several attempts to get all the details correct.
```

Comment:
I think "notoriously hard to write" is over the top. Perhaps that they are challenging to keep in sync with the code in the bundle.